src/fastDeepEqualJson.ts

```diff
@@ -0,0 +1,58 @@
+const isArray = Array.isArray;
+const hasProp = Object.prototype.hasOwnProperty;
+
+export default function fastDeepEqualJson(a: unknown, b: unknown): boolean {
```

Comment:
what does fastDeepEqualJson do if you supply it with two objects that are not json compatible?

Reply:
Undefined behavior. It attempts to compare them as if they were. We could add some types here, but the JSONValue type we've defined in our codebase doesn't work for this right now.